src/gsi/setuprad.f90

```diff
@@ -1180,6 +1182,22 @@ subroutine setuprad(obsLL,odiagLL,lunin,mype,aivals,stats,nchanl,nreal,nobs,&
               end do
            end if
 
+           if (abi2km .and. regional) then
+              abi2km_bc = zero
+              abi2km_bc(2) = 233.5_r_kind
+              abi2km_bc(3) = 241.7_r_kind
+              abi2km_bc(4) = 250.5_r_kind
+              pred(:,i) = zero
+              if (i>=2 .and. i<=4) then
+                 if (tb_obs(i) > 190.0_r_kind .and. tb_obs(i) < 300.0_r_kind) then
+                    pred(1,i)=1.
```

Comment:
One last quick modification, please add proper precision to real value 1. on line 1193 (i.e., 1._r_kind).

Reply:
Thanks for addressing this final update from me, @snebuda!